test/e2e_tests/utils/userActions.ts

```diff
@@ -86,20 +86,21 @@ export const sendTextMessageToConversation = async (
   expect(await pages.conversation().isMessageVisible(message)).toBeTruthy();
 };
 
-export const createGroup = async (pageManager: PageManager, conversationName: string, user: User[]) => {
-  await pageManager.webapp.pages.conversationList().clickCreateGroup();
-  await pageManager.webapp.pages.groupCreation().setGroupName(conversationName);
-  await pageManager.webapp.pages.startUI().selectUsers(user.flatMap(user => user.username));
-  await pageManager.webapp.pages.groupCreation().clickCreateGroupButton();
+type UserPages = PageManager['webapp']['pages'];
+export const createGroup = async (pages: UserPages, conversationName: string, user: User[]) => {
```

Comment:
I found it impractical to always pass the whole page manager even though only the pages are needed for creating a group / channel. So I refactored the few usages of the util